pulsar-common/src/main/proto/PulsarApi.proto

```diff
@@ -184,6 +184,7 @@ enum ProtocolVersion {
 	v13 = 13; // Schema-registry : added avro schema format for json
 	v14 = 14; // Add CommandAuthChallenge and CommandAuthResponse for mutual auth
 	          // Added Key_Shared subscription
+	v15 = 15; // Add batch lookup support
```

Comment:
depend on when this PR will be merged. if it is merged in 2.4.0, you can just add "Add batch lookup support" to v14.